### DISCUSSION TITLE
Reload Chart Widget

### DISCUSSION THREAD

**Discussion by psizer10** [filamentphp/filament#6923] [Help]
How do I go about reloading a chart widget?

I have a listener that receives an external parameter to be used in getData() and can see this is all running okay, logging the data is correct with the new parameter also but the chart will not update.

I am running this inside the listener function, but the chart doesn't change.

`$this->updateChartData();`

Am I missing something?

**Comment by pboivin** [comment]
Have you tried to trigger a `$refresh` with a custom event listener? See the last example on this page :
https://laravel-livewire.com/docs/2.x/actions#magic-actions
